src/discordcr/mappings/channel.cr

```diff
@@ -17,6 +17,63 @@ module Discord
       embeds: Array(Embed),
       pinned: {type: Bool, nilable: true}
     )
+
+    # A mention contained within a message, constructed by its type,
+    # the snowflake, and it's name if it is an emoji.
+    alias Mention = Tuple(MentionType, UInt64?, String?)
```

Comment:
Looking at this now, I think it would be better if I described a `Mention` as a `record` struct instead.